docs/run.md

```diff
@@ -11,6 +11,18 @@ Build citrea:
 make build
 ```
 
+### Prequisites
+
+In order for the sequencer node to run with the storage DB is needed.
```

Comment:
For production use cases, we leverage PostgreSQL for a few extra features in the sequencer. These features are optional, if you don't want to run them, make sure the `sequencer_config.toml` file does not contain the `db_config` altogether in order to skip using a storage DB backend such as postgres.